src/store/index.js

```diff
@@ -35,10 +35,6 @@ registerBlockType( UnsupportedBlock.name, UnsupportedBlock.settings );
 setUnknownTypeHandlerName( UnsupportedBlock.name );
 
 const initialHtml = `
-<!-- wp:image -->
```

Comment:
I wonder if there is some background info here on why we remove the "upload placeholder" case. Can you elaborate @etoledom ? Thanks!

Reply:
Ah, I see. No, that's cool, no need to reinstate it. It was just not clear because the actual code changes are on the GB repo side and I wasn't sure if we lost that feature or not. Thanks!